### DISCUSSION TITLE
Is it possible to make tree background transparent?

### DISCUSSION THREAD

**Discussion by JonathanDonald03** [nvim-tree/nvim-tree.lua#2965] [Q&A] [answered]
Hi is it possible to make the background of the tree transparent maybe I'm missing something but I can't seem to figure it out and when I toggle the tree it messes with the transparent background I've set for Neovim

**Comment by alex-courtis** [comment, marked as answer]
Please see `:help nvim-tree-highlight` to learn about highlight groups.

To see the appearance of nvim-tree and all other highlight groups: `:NvimTreeHiTest`

Take a look at `NvimTreeNormal` - it should set the background.

Please raise a [bug report](https://github.com/nvim-tree/nvim-tree.lua/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml) with a replicator if you continue to experience issues.
